### PR TITLE
Lepton35 motion

### PR DESCRIPTION
**Pull request by gferraro**
Added min and max temp thresh
Added support for lepton3.5

Still have to test if the defaults are correct

### CODE REVIEW

File: thermal-motion.go [hunk 1]
```diff
@@ -39,10 +43,35 @@ type ThermalMotion struct {
...
+		return DefaultLepton35Motion()
+	default:
+		return DefaultLeptonMotion()
+	}
+}
+
+func DefaultLepton35Motion() ThermalMotion {
+	return ThermalMotion{
+		DynamicThreshold: true,
+		TempThresh:       28000, //280 Kelvin ~ 7 degrees
```
Comment (clare) [inline thermal-motion.go:58]: Is there a reason there is no tempThreshMax here?

File: go.mod [hunk 2]
```diff
@@ -13,7 +15,14 @@ require (
 	github.com/spf13/afero v1.1.2
 	github.com/spf13/cast v1.3.0
 	github.com/spf13/viper v1.4.0
-	github.com/stretchr/testify v1.3.0
+	github.com/stretchr/testify v1.4.0
 	github.com/wawandco/fako v0.0.0-20180828010250-c36a0bc97398
+	golang.org/x/net v0.0.0-20200904194848-62affa334b73 // indirect
+	golang.org/x/text v0.3.2 // indirect
+	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/yaml.v1 v1.0.0-20140924161607-9f9df34309c0
```
Comment (clare) [inline go.mod:23]: Should we change to yaml.v2 too? Probably not worth the hassle though.

File: config_test.go [hunk 3]
```diff
@@ -118,8 +118,8 @@ func TestReadingConfigInDir(t *testing.T) {
 	assert.NoError(t, conf.Unmarshal(SecretsKey, &secrets))
 	assert.Equal(t, secretsChanges, secrets)
 
-	thermalMotion := DefaultThermalMotion()
-	thermalMotionChanges := DefaultThermalMotion()
+	thermalMotion := DefaultThermalMotion("")
+	thermalMotionChanges := DefaultThermalMotion("")
```
Comment (clare) [inline config_test.go:122]: DefaultThermalMotion(lepton.model30) would be more explicit.

**Review by clare — APPROVED** [review]
Thanks GP.   Looks good just check the maxtempthresh.

File: thermal-motion.go (hunk 1 above)
Comment (gferraro) [reply thermal-motion.go:58]: have rmeove tempthreshmax as we shouldn't need one

**Pull request merged**